### ISSUE TITLE
Links to Component pages are non-intuitive

### ISSUE THREAD

**Issue by tmtmtmtm** [poplus/home-poplus#19]
On http://poplus.org/catalogue.html you can click on the name of each component to go to the page for it — but this is non-obvious, as it's not in the list of links, but instead you need to know to click on the doesn't-look-like-a-link header for each.

**Comment by jpmckinney** [comment]
@tmtmtmtm Is this resolved in the recent redesign?

**Comment by jpmckinney** [comment]
I figure the red "Website" links are more intuitive.

**Comment by tmtmtmtm** [comment]
Yes. The links on the subheadings have now disappeared, which is slightly odd, but they all have an obvious link in their first line.

(It's somewhat annoying that there's no redirect from /catalogue.html to /components/examples/ though)

**Comment by jpmckinney** [comment]
Are we talking about the same links? I thought the issue was about these: http://poplus.org/components/current/

Redirects are in #34

**Comment by tmtmtmtm** [comment]
I tried clicking on the link at the top of this issue, but it 404ed. So I went to the home page in search of where the new equivalent page was. That involved clicking on the "Components" link at the top, and from there the most obvious candidate was the link at the very bottom for "examples of Poplus components in use right now". 

Now that I've looked at /components/current, then, yes, that looks more likely as the equivalent of the page I was raising the issue about, but in the absence of redirects, that was non-obvious.

So, yes, these red Website links are better than only being able to click the title for each — but it's still bad that we're inconsistent in how the links work on this page vs /components/examples/
